infrastructure/ci/dev-platform/build/create-sse-frontend-pipeline.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
This can be done with the more general script I'm working on without including the ARNs in the repo.  You can remove this file.

Reply:
removed pipeline script to use dev platform provided script